SEED-platform/seed
# fix order of migrations and null description

#### Any background context you want to provide?
On the production database, where columns already exist, when running the migrations the addition of the column_description fails because all the fields are NULL on creation since there is no default.

In addition, the merge with the GHG fields migration caused the order to be `add_column_description` then `add_ghg_fields` which caused an error because the GHG fields don't have descriptions.

#### What's this PR do?
* Remove the `merge` migration
* Renumber `0164_column_description` to `0165_column_description`
* Disable enforcing non-nulls, then alter column to enforce it again after the migration runs

#### How should this be manually tested?

If you have already run this migration locally, you might have to do a little db cleanup because some of these migrations will have already run. After the clean up then rerun `./manage.py migrate`.

Launch dbshell (using django)
```bash
./manage.py dbshell
```

```sql
/* Delete the column_description if it was already in the database */
ALTER TABLE seed_column DROP COLUMN column_description;

/* Do not remove the new GHG fields from the migration. But the others below can be removed
   because they will be recreated. Note that 0164 migration was renamed to 0165 to enforce order */
DELETE FROM public.django_migrations WHERE name='0164_column_column_description';
DELETE FROM public.django_migrations WHERE name='0165_column_column_description';
DELETE FROM public.django_migrations WHERE name='0165_merge_20220519_2058';
```

#### What are the relevant tickets?

#3216 

#### Screenshots (if appropriate)
